.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+---
+**Describe the bug**
+Please provide a concise, clear description of the bug, as well as any available error logs.
+
+**Please also include the following items to support reproducing the bug**
+1. compilers (with versions)
+2. Kokkos release or commit used (i.e., the sha1 number)
+3. platform and backend (e.g. Carawy and HIP-AMD, or Weaver and CUDA, OpenMP, etc)
```

Comment:
Isn't if `Caraway`? Do you really want to have names of particular (Sandia) machines here?

Reply:
> Isn't if `Caraway`? Do you really want to have names of particular (Sandia) machines here?

@masterleinad -- Ha!  Yes.  Good catch!  Early onset, temporary dementia, I'm afraid (the mind is the first thing to go).  I'll fix this up now.